Formula/felinks.rb

```diff
@@ -0,0 +1,65 @@
+class Felinks < Formula
+  desc "Text mode browser and Gemini, NNTP, FTP, Gopher, Finger, and BitTorrent client"
+  homepage "https://github.com/rkd77/elinks#readme"
+  url "https://github.com/rkd77/elinks/archive/refs/tags/v0.15.0.tar.gz"
+  sha256 "49e261fb7a6b061b51ca38b7152712caba90c8eaa6996bcf4c00fe2867a7f366"
+  license "GPL-2.0-only"
+  head "https://github.com/rkd77/elinks.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "groff" => :build
+  depends_on "libtool" => :build
+  depends_on "xmlto" => :build
+
+  depends_on "brotli"
+  depends_on "libidn"
+  depends_on "openssl@1.1"
+  depends_on "tre"
+  depends_on "zstd"
+
+  uses_from_macos "bison"
+  uses_from_macos "bzip2"
+  uses_from_macos "expat"
+  uses_from_macos "zlib"
+
+  conflicts_with "elinks", because: "both install the same binaries"
+
+  def install
+    # Parallel build issues tracked at upstream issue #47
```

Comment:
Can you make this into a full URL?

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/pull/106082/commits/8807325a4b3f59833bdbcf279ab05d68c1a809ae